### PR TITLE
Mosin Clips

### PR DESCRIPTION
**Pull request by PopeWreckoning**
## About The Pull Request
 Clips that allow the rapid reloading of The Mosin Bolt Action rifle.  An attempt to 
 give reprieve to the hand shuffle reloading process that causes many Mosin 
 marksmen to lose chances to continue performing long range overwatch.
 
## Why It's Good For The Game
  See above.  Makes Mosin's easier/quicker to reload.  Potentially adds chances to use 
  it in MQB due to clips speeding up, and requiring less direct attention during the 
  reload process.
  
## Changelog
 🆑 
 add: Added new mosin boltclips
 /:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
#9854, lets do this.

You got this. Ask for help when you have issues and you need assistance.

**Comment by PopeWreckoning** [comment]
> #9854, lets do this.
> 
> You got this. Ask for help when you have issues and you need assistance.

I thought that had been deleted is why I made this new one.

File: code/modules/projectiles/magazines/shotguns.dm [hunk 1]
```diff
@@ -62,6 +62,16 @@ one type of shotgun ammo, but I think it helps in referencing it. ~N
 	w_class = WEIGHT_CLASS_SMALL // CAN throw it in your pocket, friend.
 	icon_state_mini = "mosin"
 
+/obj/item/ammo_magazine/rifle/boltclip //Nearly 1:1 copy of above
+	name = "clip of 7.62x54mmR rifle rounds"
+	desc = "A Disposible Stripper clip filled with rifle bullets."
+	icon_state = "clip"
+	default_ammo = /datum/ammo/bullet/sniper/svd
```
Comment (TiviPlus) [inline code/modules/projectiles/magazines/shotguns.dm:69]: who tf made mosin use svd ammo suffering

**Review by TiviPlus — APPROVED** [review]
code ok idk about making it use boltclips but prolly fine

File: code/modules/projectiles/magazines/shotguns.dm (hunk 1 above)
Comment (PopeWreckoning) [reply code/modules/projectiles/magazines/shotguns.dm:69]: I assume it was the person who originally made the gun itself in the code cause that is what the gun is listed as taking.  Im just trying to make bullet holders, not change the ammo type.

**Review by Kuro020 — APPROVED** [review]

**Comment by TiviPlus** [comment]
Your CL was unclear I've manually updated it, I assume you tested this?

**Comment by PopeWreckoning** [comment]
I tested it in private server, and it seemed to work as intended.

**Pull request merged**